libsplinter/src/admin/rest_api/actix/proposals.rs

```diff
@@ -44,20 +44,20 @@ pub fn make_list_proposals_resource<PS: ProposalStore + 'static>(proposal_store:
     #[cfg(feature = "authorization")]
     {
         resource.add_method(Method::Get, CIRCUIT_READ_PERMISSION, move |r, _| {
-            list_proposals(r, web::Data::new(proposal_store.clone()))
+            list_proposals(r, web::Data::new(proposal_store.clone_boxed()))
```

Comment:
If you add the following to the trait file, you can still just call `clone()`

```
impl Clone for Box<dyn ProposalStore> {
    fn clone(&self) -> Box<dyn ProposalStore> {
        self.clone_boxed()
    }
}
```

Reply:
Since you added that you should put the clone_boxed back to just clone() as well